pkg/appStore/installedApp/read/InstalledAppReadService.go

```diff
@@ -0,0 +1,47 @@
+package read
+
+import (
+	"github.com/devtron-labs/devtron/pkg/appStore/installedApp/read/adapter"
+	"github.com/devtron-labs/devtron/pkg/appStore/installedApp/read/bean"
+)
+
+type InstalledAppReadService interface {
+	InstalledAppReadServiceEA
+	// GetInstalledAppByGitHash will return the installed app by git hash.
+	// Only delete specific details are fetched.
+	// Refer bean.InstalledAppDeleteRequest for more details.
+	GetInstalledAppByGitHash(gitHash string) (*bean.InstalledAppDeleteRequest, error)
```

Comment:
repo_url

Reply:
NOTE: 
`GetInstalledAppByGitHash(gitHash string)` : handling just with gitHash is not valid for different repositories. although the possibility is very low, but not impossible. 
This'll be a minor change from code side to update the query to verify the repository url as well. but as this is a logical change will be handling in a different ticket.

cc: @vikramdevtron